### DISCUSSION TITLE
App not installed as package conflicts with existing package

### DISCUSSION THREAD

**Discussion by GitUser8008** [nyxiereal/ReVancedPacker#25] [Q&A]
I'm not if there is a setting but after creating the revanced.apk using the "RV.Apk.Packer.1.8.exe", i get an error  "App not installed as package conflicts with existing package" when trying to install the revanced.apk.    Have you seen this before?  What did I do wrong?  My phone is not rooted.  My guess is it's trying to overwrite the youtube app that came with the phone but I don't know what to do to fix or repoint the install directory.

I was following the guide from you wiki page.

https://github.com/xemulat/ReVancedPacker/wiki/How-to-use-this-program-(guide)

**Comment by rootajoe11** [comment]
Go to Settings of your phone and look for an option that lists all applications in your phone and try to find whether the application that you are trying to install is already listed. If it is, then uninstall it from there.

Another common reason for the App not installed error could be that there is not enough free memory on your device's internal storage. Most users think that the size of the [apk ](https://apktecch.com/) file is the actual size of the app. But this is not the case. Actually the apk file is a packaged version of the application itself.
